src/Resource/Payment.php

```diff
@@ -581,11 +581,12 @@ public function setOrder(Orders $order)
     /**
      * Turns on a delay on credit card payment capture (pre-authorization).
      *
+     * @param $value bool
      * @return $this
      */
-    public function setDelayCapture()
+    public function setDelayCapture($value = true)
     {
-        $this->data->delayCapture = true;
+        $this->data->delayCapture = $value;
```

Comment:
@lucascolette is this still setting de default to be `true`?

Is the difference now the fact that you can also pass the value `false` to the function?